CONTRIBUTING.md

```diff
@@ -85,19 +85,22 @@ You can access the app [localhost:3000](http://localhost:3000). Whatever you cha
   * If not, please first leave a comment on how you want to implement it and wait for contributors' feedback
 
 ### Commits
-* Commit title should be short and descriptive
-  * title (or summary line) is the first line of the commit message
-  * that says what the commit is doing
-  * in no more than 50 characters
-  * starting with a word like 'Fix' or 'Add' or 'Change'
-  * **without** a period (.) at the end
-  * followed by a blank line
-* Commit messages are
-  * up to 72 characters
-  * with break lines
-* Reference the issue(s) the commit closes
-  * https://help.github.com/articles/closing-issues-via-commit-messages
-  * If you haven't done so since the beginning, you should reference the issue when you squash your commits
+* The commit title should be short and descriptive, and it should follow these rules:
+  * It should be the first line of the commit message
+  * It explains what the commit does in the imperative mood (Fix foobar)
+  * It's no longer than 50 characters
+  * It starts with a word like 'Fix' or 'Add' or 'Change'
+  * It does not end with a period (.)
+  * It is followed by a blank line
+  * It should contain more than only the reference to the issue
+
+Read this [article](https://chris.beams.io/posts/git-commit/) for more information
+* The commit message should follow these rules:
+  * It should explain why the change is made
+  * It should not describe how the change is made, code is generally self-explanatory
+  * It's not longer than 72 characters
+  * It should contain line breaks
```

Comment:
So that the 1st case does not occur, and we end up with commit messages like the 2nd commit.

commit 21052b2639e471628b0db52affd61198a306890a
Author: Stella Rouzi <differentreality@gmail.com>
Date:   Tue Mar 21 11:49:55 2017 +0200

    This is a very very big commit message that I am writing here to test something I need, because I want to test what happens with a very very long commit message, and if I have to scroll horizontally in order to view it, or not. and if I do, I want to include this into our styling guide

commit 50f097811c88f3220701e958446b53a96a7ce9c5
Author: Stella Rouzi <differentreality@gmail.com>
Date:   Tue Jun 28 18:16:06 2016 +0300

    Add surveys; custom generated by admins
    
    Available during registration or after conference



![screenshot from 2017-03-21 11-51-36](https://cloud.githubusercontent.com/assets/4322737/24141654/c82db3ba-0e2c-11e7-99a0-b0e54ec0679a.png)